rwgame/State.cpp

```diff
@@ -4,6 +4,39 @@
 // This serves as the "initial" camera position.
 ViewCamera defaultView({-250.f, -550.f, 75.f}, glm::angleAxis(glm::radians(5.f), glm::vec3(0.f, 1.f, 0.f)));
 
+
+void State::handleEvent(const SDL_Event& e)
+{
+	auto m = getCurrentMenu();
+	if(!m) return;
+
+	switch(e.type) {
+	case SDL_MOUSEBUTTONUP:
+		if (e.button.button == SDL_BUTTON_LEFT)
+			m->click(e.button.x, e.button.y);
+		break;
+
+	case SDL_MOUSEMOTION:
+		m->hover(e.motion.x, e.motion.y);
+		break;
+
+	case SDL_KEYUP:
```

Comment:
This makes it no longer possible to hold the arrow keys to navigate the menus quickly. Is there any particular reason to not use KEYDOWN?